pom.xml

```diff
@@ -278,6 +278,11 @@
       <version>1.10.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jdt</groupId>
+      <artifactId>org.eclipse.jdt.annotation</artifactId>
+      <version>2.1.0</version>
```

Comment:
We don't need a new dependency just for a test.
Please remove this.

Reply:
@sergileon , you introduce compilation dependency for whole project - it NOT ok.
even you make a scope - "test" it will not be ok as a reason is just to keep out inputs compilable.
Please recreate annotation in our resources by almost "copy-paste"(minimized).